src/test/java/automation_workshop/GalleryTest.java

```diff
@@ -8,7 +8,7 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.openqa.selenium.support.PageFactory;
-import pages.HomePageFS;
+import pages.hero;
```

Comment:
As mentioned above, the Hero Class Name should be capitalized... looks a bit weird :(

Reply:
Done